{"name": "Попытка авторизации с неверными данными", "status": "passed", "steps": [{"name": "Авторизация пользователя", "status": "passed", "parameters": [{"name": "email", "value": "'wrong@email.com'"}, {"name": "password", "value": "'N349HI)bO&'"}], "start": 1729158593693, "stop": 1729158593993}, {"name": "Проверяем код ответа. Код ответа - 401", "status": "passed", "start": 1729158593993, "stop": 1729158593993}, {"name": "Проверяем значение поля 'message' в ответе (email or password are incorrect)", "status": "passed", "start": 1729158593994, "stop": 1729158593994}], "parameters": [{"name": "email", "value": "'wrong@email.com'"}, {"name": "password", "value": "'correctpassword'"}], "start": 1729158593693, "stop": 1729158593994, "uuid": "b2cea901-fd41-4952-8ad6-cb66b9cffca1", "historyId": "832c25a83148f922513088a4ab91a401", "testCaseId": "fbeae77312463979bb58e896f211518f", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_invalid_credentials", "labels": [{"name": "feature", "value": "Авторизация пользователя"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "minergenon"}, {"name": "thread", "value": "1308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}